util/a2e/headers/ctype.h

```diff
@@ -33,6 +33,13 @@
  * ===========================================================================
  */
 
+#if !defined(OMR_CTYPE_H)
+#define OMR_CTYPE_H
+
+#if defined(J9ZOS390)
+#define _AE_BIMODAL
+#endif /* defined(J9ZOS390) */
```

Comment:
This is at best redundant (it should be defined by the make system) or may be viewed by the compiler as a redefinition (error).